packages/salesforcedx-vscode-lwc/README.md

```diff
@@ -1,5 +1,56 @@
+This extension provides code-editing features for the Lightning Web Components programming model, which is part of the Lightning Component framework. It uses the default HTML language server from VS Code to provide syntax highlighting, code completion, an outline view of your files, and a Salesforce Lightning Design System (SLDS) linter.
 
-We provide salesforce-vscode-lwc to selected customers through a pilot program that requires agreement to specific terms and conditions. salesforce-vscode-lwc is subject to change and isn’t generally available unless or until Salesforce announces its general availability in documentation or in press releases or public statements. We can’t guarantee general availability within any particular time frame or at all. Make your purchase decisions only on the basis of generally available products and features.
+**Currently, this extension is not included in the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode).**
+
+## Prerequisites
+
+Before you set up this extension
+
+* Make sure that you have [Visual Studio Code](https://code.visualstudio.com/download) v1.26 or later.
+
+* Install the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) to get all the required dependencies.
+
+## Features Provided by This Extension
+
+* Notification of HTML and JavaScript file errors or compiler warnings
+
+* ESLint configuration for Lightning web components
+    * Errors and warnings appear in JavaScript files
+    * Hover over code to display warning messages
+    * Click the displayed message for available code actions
+  
+* Auto-completion for resources in JavaScript files
+    * Static resources
+    * Custom label imports
+    * Lightning web components imports from `engine`
+
+* Auto-completion for resources in HTML files
+    * Tags and attributes for standard `lightning` namespace Lightning web components
+    * Tags and attributes for custom `c` namespace Lightning web components
+    * Lightning web components directives in related HTML files
+  
+* Help documentation when you hover over standard `lightning` namespace Lightning web components or attributes
+
+* Click navigation from HTML files to the main JavaScript file for custom `c` namespace Lightning web components and attributes
+  
+
+## Bugs and Feedback
+
+To report issues with Salesforce Extensions for VS Code, open a [bug on GitHub](https://github.com/forcedotcom/salesforcedx-vscode/issues/new?template=Bug_report.md). If you would like to suggest a feature, create a [feature request on Github](https://github.com/forcedotcom/salesforcedx-vscode/issues/new?template=Feature_request.md).
+
+## Resources
+
+- Doc: [Lightning Web Components Developer Guide](http://developer.salesfore.com/docs/component-library/documentation/lwc)
```

Comment:
This link doesn't yet work for me. When does this go live?

Reply:
This one goes live Dec 17th. Maybe I should remove it now. Can we add it in a week?

Reply:
Luis just mentioned this will go live earlier. I think I'll add text saying "available after December 17th" OK?